invoker/rh-changes/pom.xml

```diff
@@ -82,6 +82,12 @@
                 <version>${redhat.che.version}</version>
                 <type>war</type>
             </dependency>
+            <dependency>
+                <groupId>com.redhat.che</groupId>
+                <artifactId>fabric8-ide-stacks</artifactId>
+                <version>${redhat.che.version}</version>
```

Comment:
Is there a reason why the stacks version should be `${redhat.che.version}` ?
I mean `${redhat.che.version}` is following the upstream Che version with a suffix (`fabric8` or `fabric8-without-dashboard`). It has sense for the RH components that *derive from* the upstream components (assemblies, dashboard) but with modifications / additions.
On the contrary, components that exist on their own (such as  additional RH plugins) can have their own versioning.  How are these stacks going to be managed ? are they going to be regularly updated from their upstream counterparts, or are they going to have their own life ?